src/main/java/de/hysky/skyblocker/skyblock/item/tooltip/adders/TooltipAdder.java

```diff
@@ -0,0 +1,49 @@
+package de.hysky.skyblocker.skyblock.item.tooltip.adders;
+
+import de.hysky.skyblocker.skyblock.ChestValue;
+import de.hysky.skyblocker.skyblock.item.tooltip.TooltipManager;
+import net.minecraft.screen.slot.Slot;
+import net.minecraft.text.Text;
+
+import java.util.List;
+import java.util.regex.Pattern;
+
+/**
+ * Extend this class and add it to {@link TooltipManager#adders} to add additional text to tooltips.
+ */
+public abstract class TooltipAdder {
```

Comment:
This is in the `adders` package, but `SlotTextAdder` is not in its `adders` package. I would not put it in the `adders` package.